forest/bonsai/simulate_gps_data.py

```diff
@@ -6,20 +6,26 @@
 import datetime
 from dataclasses import dataclass
 from enum import Enum
+import json
 import os
+import re
+import requests
```

Comment:
`requests` is a 3rd party module so should go into the next section. Also, need to add `types-requests` to `requirements.txt` for `mypy` checks to pass.